HISTORY.rst

```diff
@@ -6,6 +6,8 @@ Changelog
 
 **API Changes**
 
+- Throw ``PriorityLoop`` when inserting or reprioritizing a stream that
```

Comment:
Somebody who nitpicks things would point out that the rest of the changelog uses the British spelling or “reprioritise”. Do we know anybody like that?

😛